pyproject.toml

```diff
@@ -11,16 +11,21 @@ keywords = [
 license = "Apache-2.0"
 
 [tool.poetry.dependencies]
-python = "<3.12,>=3.7.1"
-singer-sdk = { version="^0.27.0" }
-fs-s3fs = { version = "^1.1.1", optional = true }
+python = "<3.12,>=3.8"
+singer-sdk = { version="^0.28.0" }
+s3fs = { version = "^2023.5.0", optional = true}
+fsspec = "^2023.5.0"
+ruff = "^0.0.270"
```

Comment:
Linters we put under dev dependencies as they aren't needed to run the app. (black as well)

Reply:
I've moved them to be dev dependencies.